Dockerfile

```diff
@@ -12,8 +17,13 @@ RUN mkdir /app
 WORKDIR /app
 
 COPY "$PWD/fixtures" /app/fixtures
-COPY "$PWD/docker-entrypoint.sh" /app/docker-entrypoint.sh
 COPY "$PWD/joplin" /app/joplin
 
-ENTRYPOINT ["./docker-entrypoint.sh"]
+COPY "$PWD/migrate-load-data.sh" /app/migrate-load-data.sh
+RUN LOAD_DATA=on ./migrate-load-data.sh
```

Comment:
By doing this as a RUN command instead of having it in the ENTRYPOINT, it means that when we boot up the built image on heroku it only boots up wagtail instead of migrating and loading data.

Reply:
So does this mean that we were previously blowing away prod data each deploy and we won't be doing that anymore?